apps/src/templates/PaneHeader.jsx

```diff
@@ -157,22 +160,23 @@ export const PaneButton = Radium(function(props) {
   }
 
   return (
-    <div
+    <button
       className={divClassNames}
-      role="button"
+      type="button"
```

Comment:
nit: can we rename the classname from `divClassNames` since this is no longer a div?